web/cypress/e2e/account-settings.cy.ts

```diff
@@ -684,4 +684,222 @@ describe('Account Settings Page', () => {
+      cy.get('[data-testid="copy-token-button"]')
+        .find('input')
+        .should('have.value', 'fake-token-code-12345');
+
+      // Test Kubernetes Secret tab
+      cy.contains('Kubernetes Secret').click();
+      cy.contains('Step 1: Create secret YAML file').should('exist');
+      cy.contains('apiVersion: v1').should('exist');
+      cy.contains('kind: Secret').should('exist');
+
+      // Test rkt Configuration tab
+      cy.contains('rkt Configuration').click();
+      cy.contains('Step 1: Create rkt configuration file').should('exist');
+      cy.contains('rktKind').should('exist');
+
+      // Test Podman Login tab
+      cy.contains('Podman Login').click();
+      cy.contains('Run podman login command').should('exist');
+      cy.contains('podman login').should('exist');
+
+      // Test Docker Login tab
+      cy.contains('Docker Login').click();
+      cy.contains('Run docker login command').should('exist');
+      cy.contains('docker login').should('exist');
+
+      // Test Docker Configuration tab
+      cy.contains('Docker Configuration').click();
+      cy.contains('Step 1: Create Docker config file').should('exist');
+      cy.contains('This will').should('exist');
+      cy.contains('overwrite').should('exist');
+    });
+
+    cy.get('[data-testid="create-token-close"]').click();
+    cy.get('[data-testid="create-token-modal"]').should('not.exist');
+  });
+
+  it('Clickable Token Titles - View Token Details', () => {
+    // Mock existing tokens
+    cy.intercept('GET', '/api/v1/user/apptoken', {
+      body: {
+        tokens: [
+          {
+            uuid: 'token1-uuid',
+            title: 'Clickable Token',
+            last_accessed: '2024-01-15T10:30:00Z',
+            created: '2024-01-01T09:00:00Z',
+            expiration: '2024-12-31T23:59:59Z',
+          },
+        ],
+        only_expiring: false,
+      },
+    }).as('getTokens');
+
+    cy.visit('/organization/user1?tab=Settings');
+    cy.contains('CLI configuration').click();
+    cy.wait('@getTokens');
+
+    // Token title should be a clickable link
+    cy.get('table')
+      .last()
+      .within(() => {
+        cy.contains('button', 'Clickable Token').should('exist');
+      });
+
+    // Click on token title
+    cy.contains('button', 'Clickable Token').click();
+
+    // View token modal should appear
+    cy.contains('Application Token: Clickable Token').should('be.visible');
+
+    // Verify modal shows token details
+    cy.contains('Title').should('exist');
+    cy.contains('Clickable Token').should('exist');
+    cy.contains('Created').should('exist');
+    cy.contains('1/1/2024').should('exist');
+    cy.contains('Last Accessed').should('exist');
+    cy.contains('1/15/2024').should('exist');
+    cy.contains('Expiration').should('exist');
+    cy.contains('12/31/2024').should('exist');
+
+    // Verify informational alert
+    cy.contains(
+      'The token code is only shown once when the token is created',
```

Comment:
This behavior is different than the old UI where we can view the token after creation - do we want to change that?